Distributions/OpacitySource/wlOpacityTableIOModuleHDF.f90

```diff
@@ -394,6 +424,18 @@ SUBROUTINE ReadOpacityTableHDF &
       nMom_Pair  = 0
     END IF
 
+    IF( PRESENT( FileName_Brem_Option ) &
+        .AND. ( LEN( FileName_Brem_Option ) > 1 ) )THEN
+      ReadOpacity(iBrem) = .TRUE.
+      FileName   (iBrem) = TRIM( FileName_Brem_Option )
+      nOpac_Brem = 1
```

Comment:
Will `nOpac_Brem` and `nMom_Brem` always be 1?

If not, maybe some comment here describing why they are set to 1.

Reply:
They are for now, but probably not necessarily always.